docs/connecting-to-ethereum/setting-up-wallet.md

```diff
@@ -0,0 +1,54 @@
+---
+id: setting-up-wallet
+title: Setting up your wallet
+---
+
+<!-- TODO: Add intro on wallets -->
+
+## Connecting your wallet
+
+To connect your wallet, just head to the wallet tab in the sidebar of your Radicle Upstream client and click connect.
+
+![Wallet][cw]
+
+When clicking **Connect your wallet** you'll be prompted to connect using a [WalletConnect supported wallet][wc].
+
+![Wallet Connect][qr]
+
+When scanning the QR code you'll be prompted in your wallet to "Connect to Radicle Upstream". Once you've done that, the Upstream app ui will update to show the balance of your connected wallet.
+
+![Wallet Page][wp]
+
+Your wallet is now connected. The next step is to link your Radicle ID to your Ethereum address.
+
+## Linking your Radicle ID
+
+ow that your wallet is connected, the next step is to link your Radicle ID to your Ethereum address. This will mean that we can find your Ethereum address based on your Radicle ID and vice versa.
```

Comment:
```suggestion
Now that your wallet is connected, the next step is to link your Radicle ID to your Ethereum address. This will mean that we can find your Ethereum address based on your Radicle ID and vice versa.
```